armi/runLog.py

```diff
@@ -310,20 +310,29 @@ def concatenateLogs(logDir=None):
 
     info("Concatenating {0} log files".format(len(stdoutFiles)))
 
+    combinedLogName = os.path.join(logDir, "armi-mpi-workers.log")
+    with open(combinedLogName, "w") as workerLog:
+        workerLog.write(
+            "\n{0} CONCATENATED WORKER LOG FILES {1}\n".format("-" * 10, "-" * 10)
+        )
+
     for stdoutName in stdoutFiles:
         # NOTE: If the log file name format changes, this will need to change.
         rank = int(stdoutName.split(".")[-2])
-
-        # first, print the log messages for a child process
         with open(stdoutName, "r") as logFile:
```

Comment:
stdoutName is just here, so i think you can get the name from that. you will need to move the above

{0} CONCATENATED WORKER LOG FILES 

into the loop since the overall code allows for multiplied groups of stdouts, but shouldn't be a big deal.

Reply:
`stdoutName` is not the `caseTitle` here. It's something like `ARMI.0001.stdout`, `ARMI.0002.stdout`, etc.